website/docs/package.json

```diff
@@ -17,7 +17,7 @@
 		"lint": "prettier --plugin-search-dir . --check .",
 		"format": "prettier --plugin-search-dir . --write .",
```

Comment:
From what I can see locally, we make no use of **lint** or **format**. Prettifying is done on the commit hook anyway. So could you remove these two lint and format scripts as well as the prettier dependency down below, so we don't have to maintain its versions in the future

Reply:
Done